lavamoat/webpack/policy.json

```diff
@@ -0,0 +1,1383 @@
+{
```

Comment:
This file is generated by lavamoat automatically (because we have `generatePolicy: true` in our plugin config). All manual changes are done in the `policy-override.json`